wgpu/utils/imgui/imgui_renderer.py

```diff
@@ -144,8 +144,8 @@ def render(self):
 
         imgui.set_current_context(self.imgui_context)
 
-        pixel_ratio = self._canvas_context.canvas.get_pixel_ratio()
-        lsize = self._canvas_context.canvas.get_logical_size()
+        pixel_ratio = self._canvas_context.pixel_ratio
+        lsize = self._canvas_context.logical_size
```

Comment:
we can certainly add an if statement here too. but i'm always hesitant to adding "if statements' in the runtime path. I would probably add it around the entire `def render` so that there would be 2 versions of the function.